.azure-pipelines/azure-pipelines-merge.yml

```diff
@@ -204,7 +204,7 @@ jobs:
 #    vmImage: 'macOS-10.13'
 #
 #  steps:
-#  - script: git checkout release-0.16.0
```

Comment:
These were commented out coz of some breaking change earlier. Generally we file a tracking issue to uncomment these once release is done but I can't find it. 

Can you uncomment these as part of this PR. 

There is another section in pr.yaml also.

Reply:
Okay will do that!

Reply:
Done in the next commit!